### PR TITLE
Fixes #7653: Sorting by hostname in nodes list doesn't work

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/7653

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
PR updated

**Comment by VinceMacBuche** [comment]
PR updated

**Comment by VinceMacBuche** [comment]
PR updated

**Comment by fanf** [comment]
why targetting 3.1 ? Isn't there the same pb in 3.0 (same datable version ?)

File: rudder-web/src/main/webapp/templates-hidden/common-layout.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <script type="text/javascript" src="/javascript/jstree/jquery.jstree.min.js" id="jstree"></script>
     <!-- using minified version of that one leads to errors "DataTables warning (table id ...: 
          Added data (size undefined) does not match known number of columns" errors -->
-    <script type="text/javascript" src="/javascript/datatables/js/jquery.dataTables.js"></script>
+    <script type="text/javascript" src="/javascript/datatables/js/jquery.dataTables.min.js"></script>
```
Comment (fanf) [inline rudder-web/src/main/webapp/templates-hidden/common-layout.html]: bad uri, should be: /javascript/datatables/datatables.min.js

**Comment by fanf** [comment]
Appart from that, I didn't reproduced the bug when sorting by compliance.

**Comment by VinceMacBuche** [comment]
PR updated

**Comment by VinceMacBuche** [comment]
PR replaced by https://github.com/Normation/rudder/pull/1064

**Pull request closed without merging**